MetaMask/metamask-extension
# MCv1 Permissions: Site Connections

## **Description**
This PR adds the ability to use Accounts in the Avatar Group component, and adds a new Account Group in the Connection List Item for non-snap items. TODO: tooltips (DS creating new component), and per-dapp network badges.


[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23080?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1980

## **Manual testing steps**

1. Open Metamask > All Permissions
2. Verify that Account avatar groups are limited to 5 showing per dapp

## **Screenshots/Recordings**



### **Before**


<img width="356" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/9cfe77b1-1927-434a-beb4-fd36ff44c676">


### **After**


<img width="360" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/84367c90-87d5-411e-930f-89523e20c1f5">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
